sdk/schemaregistry/azure-data-schemaregistry/src/main/java/com/azure/data/schemaregistry/implementation/models/SchemasGetByIdHeaders.java

```diff
@@ -197,7 +211,7 @@ public SchemasGetByIdHeaders setLocation(String location) {
      *
      * @return the contentType value.
      */
-    public SchemaFormatImpl getContentType() {
+    public SchemaFormat getContentType() {
```

Comment:
Curious why the change from SchemaFormatImpl -> SchemaFormat? I think there is a collision with the public-facing one so we have to qualify it. :/

Reply:
This is done by Autorest, we can revert this if its a breaking change ?